mesonbuild/compilers/compilers.py

```diff
@@ -193,6 +200,13 @@ def is_library(fname):
                         'minsize': [],
                         }
 
+arm_buildtype_args = {'plain': [],
+                        'debug': ['-O0', '--debug'],
+                        'debugoptimized': ['-O1', '--debug'],
+                        'release': ['-O3', '-Otime'],
+                        'minsize': ['-O3', '-Ospace'],
+                        }
```

Comment:
__[Flake8]__

[E124] closing bracket does not match visual indentation


<!-- Comment by [SideCI](https://sideci.com) -->